assets/scss/partials/components/_notice.scss

```diff
@@ -0,0 +1,51 @@
+.notice {
+  padding-left: 0;
+  padding-right: 0;
+  width: 100%;
```

Comment:
I think these lines are unnecessary on the `<div>` where it would be applied. Please clear me up if it is.